pact/fixed-quote-royalty-policy.pact

```diff
@@ -87,10 +89,12 @@
             (royalty-rate:decimal (at 'royalty-rate spec))
             (creator-details:object (fungible::details creator ))
             )
-      (fungible::enforce-unit royalty-rate)
+      (enforce (>= min-amount 0.0) "Invalid min-amount")
+      (enforce (>= max-supply 0.0) "Invalid max-supply")
       (enforce (=
         (at 'guard creator-details) creator-guard)
         "Creator guard does not match")
+      (fungible::enforce-unit royalty-rate)
```

Comment:
I don't think this is necessary ... in any case it won't stop precision problems below.